v3/src/components/graph/adornments/movable-line.tsx

```diff
@@ -185,8 +282,15 @@ export const MovableLine = (props: {
   }, [])
```

Comment:
Address the rule-of-hooks warning. If this is intended to run only on mount, then the warning can simply be disabled along with a comment as to why.